components/elm/src/data_types/TopounitType.F90

```diff
@@ -10,7 +10,7 @@ module TopounitType
   use shr_kind_mod   , only : r8 => shr_kind_r8
   use shr_infnan_mod , only : nan => shr_infnan_nan, assignment(=)
   use shr_log_mod    , only : errMsg => shr_log_errMsg
-  use abortutils     , only : endrun
+  ! use abortutils     , only : endrun
```

Comment:
Remove commented code

Reply:
@thorntonpe I have removed commented code in my latest commit. Thanks.